python-sdk/src/astro/databases/aws/redshift.py

```diff
@@ -292,7 +292,9 @@ def merge_table(
             for statement in statements:
                 cursor.execute(statement)
 
-    def is_native_load_file_available(self, source_file: File, target_table: BaseTable) -> bool:
+    def is_native_load_file_available(
+        self, source_file: File, target_table: BaseTable  # skipcq PYL-W0613
```

Comment:
a override of base class method

Reply:
why?

Reply:
never mind got it. https://deepsource.io/gh/mindcurry/UniBorg/issue/PYL-W0613/occurrences